students/briggsm/session05/mailroom3.py

```diff
@@ -0,0 +1,109 @@
+'''Session 5 Mailroom3
```

Comment:
I think I mostly reviewed this in its earlier version.

let me know if there's something new you want me to look at.